test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua

```diff
@@ -0,0 +1,56 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when WEB_VIEW AppHmiType is absent in application policies
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
```

Comment:
```suggestion
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-82aaf8dae27f1b695f9cc2b4592ceba3R24)